src/platform/plugins/shared/home/public/application/components/sample_data/index.tsx

```diff
@@ -51,11 +53,23 @@ export function SampleDataCard({ urlBasePath, onDecline, onConfirm }: Props) {
       }
       footer={
         <footer>
-          <EuiButton fill className="homWelcome__footerAction" onClick={onConfirm}>
+          <EuiButton
+            fill
+            css={({ euiTheme }: UseEuiTheme) =>
+              css({
+                marginRight: euiTheme.size.s,
+              })
+            }
+            onClick={onConfirm}
+          >
             <FormattedMessage id="home.tryButtonLabel" defaultMessage="Add integrations" />
           </EuiButton>
           <EuiButtonEmpty
-            className="homWelcome__footerAction"
+            css={({ euiTheme }: UseEuiTheme) =>
+              css({
+                marginRight: euiTheme.size.s,
+              })
+            }
```

Comment:
We could consolidate these two definitions into a single declaration that are then referenced for brevity, also it means only one style will be generated instead of two. Maybe something like so; 

```suggestion
const footerAction = useCallback(({ euiTheme }: UseEuiTheme) => {
	return  css({
		marginRight: euiTheme.size.s,
	})
}, [euiTheme]);

...
<EuiButtonEmpty fill css={footerAction} ...


```